### ISSUE TITLE
dataset scroll lock jumps irregularly

### ISSUE THREAD

**Issue by MortenHofft** [gbif/portal16#28]
The fixed position overview menu for datasets is unstable. It jumps in some cases - not exactly clear which. See screen shot of examples. Also be aware that position fixed can act weird on mobile safari when input fields get focus. So this should also be tested or at least considered.

![screen shot 2016-06-20 at 12 15 39](https://cloud.githubusercontent.com/assets/6759267/16453268/8e7d8f96-3e0c-11e6-9d14-48cbd6f55ae8.png)
![screen shot 2016-06-20 at 12 15 25](https://cloud.githubusercontent.com/assets/6759267/16453270/8ea2f16e-3e0c-11e6-8d91-f988c623bd13.png)

**Comment by MortenHofft** [comment]
Other parts o the styling needs another parse I see
![screen shot 2016-07-06 at 09 32 14](https://cloud.githubusercontent.com/assets/6759267/16610650/d5d67fec-435c-11e6-9684-633ea8dd7267.png)

**Comment by burkeker** [comment]
Noted. Though would be nice to know how to reproduce the first two cases. I've tried different browsers at desktop but haven't been able to reproduce it.

**Comment by MortenHofft** [comment]
Sorry about that Burke - it is indeed a lacking bug report. I simply noticed oddities when i played around with it and took screenshots. I would guess that they are small issues to fix.

I do not know your implementation, but looking at the pictures I would guess that you're positioning event hasn't been fired. Probably due to changing screen sizes. I'm guessing that you have a *scroll* event that updates some classes. When *resizing* you probably need to fire the same event.

As for the grey bar across the text area, it is - I suspect - reproducible by hovering over the "Bibliographic citations" bullet.

I used chrome, but I would expect it to be the same in all other browsers. To reproduce you probably just need to change window size and scroll around a bit.

Hope that helps
